src/screens/UserPortal/Settings/Settings.tsx

```
@@ -270,6 +268,70 @@
+                        type="file"
+                        className={styles.cardControl}
+                        data-testid="fileInput"
+                        multiple={false}
+                        ref={fileInputRef}
+                        onChange={
+                          /* istanbul ignore next */
+                          async (
+                            e: React.ChangeEvent<HTMLInputElement>,
+                          ): Promise<void> => {
+                            const target = e.target as HTMLInputElement;
+                            const file = target.files && target.files[0];
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)